src/js/views/maps/viewfinder/ZoomPresetsListView.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+
+define(
+  [
+    'underscore',
+    'backbone',
+    'views/maps/viewfinder/ZoomPresetView',
+  ],
+  (_, Backbone, ZoomPresetView) => {
+    // The base classname to use for this View's template elements.
+    const BASE_CLASS = 'viewfinder-zoom-presets';
+
+    /**
+     * @class ZoomPresetsListView
+     * @classdesc Allow user to zoom to a preset location with certain data
+     * layers enabled.
+     * @classcategory Views/Maps/Viewfinder
+     * @name ZoomPresetsListView
+     * @extends Backbone.View
+     * @screenshot views/maps/viewfinder/ZoomPresetsListView.png
+     * @since x.x.x
+     * @constructs ZoomPresetsListView
+     */
+    var ZoomPresetsListView = Backbone.View.extend(
+    /** @lends ZoomPresetsListView.prototype */ {
+        /**
+         * The type of View this is
+         * @type {string}
+         */
+        type: 'ZoomPresetsListView',
+
+        /** @inheritdoc */
+        className: BASE_CLASS,
+
+        /**
+         * @typedef {Object} ZoomPresetsListViewOptions
+         * @property {ViewfinderModel} The model associated with the parent view.
```

Comment:
Is it worth passing in zoomPresets and selectZoomPreset directly to decouple it from Viewfinder?

Reply:
Oh yeah I like that, alleviates the need for me to check in an unimplemented function in viewfinder model as well, thanks